packages/styles/scss/patterns/blocks/content-block-simple/_content-block-simple.scss

```diff
@@ -0,0 +1,21 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+@import '../../../globals/imports';
+@import '../../sub-patterns/card/index';
+@import '../../../components/link-with-icon/link-with-icon';
+
+@mixin content-block-simple {
+  .#{$prefix}--content-block-simple {
+    &__content {
+      padding-bottom: $carbon--spacing-07;
```

Comment:
Is there a reason to apply it here instead of applying it to `ContentBlock`?

Reply:
This is a conversation I want to have with the team in terms of where spacing should be applied when nesting patterns. Right now we have block, group, and item levels all specifying their own spacing, and when used in another container it doesn't display correctly. This is even more apparent for optional items, e.g. if a heading with `padding-top` is optional, and is not present in a content block, the padding goes away with it and the above/below elements could now be crashing.

Reply:
We need to take a look at this across the board. This is on my to do for this sprint. Agreed it is a problem, but the solution I feel is to fix it at the sub-pattern level instead of having these one offs.

Reply:
I agree. But with the total inconsistency from one block/group/component to the next, it is impossible not to have them in some cases. Will leave it as is for now and let's get an issue where we fix globally.